src/main/java/competition/subsystems/shooterwheel/ShooterWheelSubsystem.java

```diff
@@ -36,7 +37,9 @@ public void spin () {
 
     public void setPower(double power) {
         if(contract.isShooterWheelReady()){
-            shooterWheelMaster.simpleSet(power);
+        	neoMasterMotor.set(power);
+            neoFollowerMotor.set(power);
```

Comment:
You don't need to ever command the follower motor - it automatically follows the leader.